cdap-docs/reference-manual/source/http-restful-api/metrics.rst

```diff
@@ -638,7 +721,7 @@ difference calculated between the start and end times:
 
 Example::
 
-  POST '<base-url>/metrics/query?context=namespace.default.app.CountRandom&
+  POST '<base-url>/metrics/query?tag=namespace:default&tag=app:CountRandom&
```

Comment:
L#728 the description is incorrect, you are saying the resolution is in seconds, while it should be in minutes